pos_payment_restriction/__manifest__.py

```diff
@@ -9,7 +9,7 @@
     "license": "AGPL-3",
     "development_status": "Beta",
     "maintainers": ["rousseldenis"],
-    "category": "Point of Sale",
+    "category": "Sales/Point Of Sale",
```

Comment:
I'm not aware of conventions regarding categories, and i never know what to write.
Can you tell me where it is? 

Thanks !

Reply:
> I'm not aware of conventions regarding categories, and i never know what to write.
> Can you tell me where it is?
> 
> Thanks !

The problem I just faced is during migration if categories changed.
Former one : https://github.com/odoo/odoo/blob/9.0/addons/point_of_sale/__openerp__.py#L7
New one: https://github.com/odoo/odoo/blob/13.0/addons/point_of_sale/__manifest__.py#L7

I have a foreign key error on module category, solved changing manifest like this.

For info, OCA shop contains Odoo aggregated categories through core modules : 
![image](https://user-images.githubusercontent.com/19529533/107068070-6e8cf980-67e0-11eb-8241-c8506d5c6163.png)

I see now they are not sync since shop development. Maybe something to do.

To summarize, it's always recommended to fit OCA shop categories in order to be under the good filter.